qdrant-landing/content/documentation/guides/distributed_deployment.md

```diff
@@ -820,6 +848,17 @@ The service will download the specified snapshot of the collection and recover s
 
 Once all shards of the collection are recovered, the collection will become operational again.
 
+### Temporary node failure
+
+If properly configured, running Qdrant in distributed mode can make your cluster resistent to outages when one node is down temporarily.
```

Comment:
Does this still work in a "one-node" configuration? If not:

```suggestion
If you have a multi-node configuration, you can run Qdrant in distributed mode. This can make your cluster resistant to outages when one node fails.
```